### PR TITLE
Chore/docker and tests

### PR DESCRIPTION
**Pull request by clettenberg**
Add basic Docker setup and run tests in CI

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Specify the base image
+FROM node:14
```
Comment (zenlex) [inline Dockerfile:2]: Can we bump this to 20 since it's LTS now?

**Review by zenlex — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (clettenberg) [reply Dockerfile:2]: yep!

Comment (clettenberg) [reply Dockerfile:2]: Should i just make it `node:lts`? Or do you prefer a pinned version

Comment (zenlex) [reply Dockerfile]: I was too slow - I prefer the pin haha. Also I think we can drop node-fetch and use native fetch after 18+ :)

**Comment by zenlex** [comment]
Nice work @clettenberg ! I went ahead and pinned node and bumped/deleted the relevant stuff. Thanks again for all your help!

**Pull request merged**